apps/dummyapp/widget/hello.jsx

```diff
@@ -1 +1,7 @@
+const ss = Social.get("dummydata/dummy/dummy")
```

Comment:
I had to change this to "dummy/dummydata/dummy/dummy" in order to get it to work, was missing "accountId"

Reply:
You're right. Fixed!